app/models/setting/provisioning.rb

```diff
@@ -14,6 +14,7 @@ def self.load_defaults
         self.set('safemode_render', N_("Enable safe mode config templates rendering (recommended)"), true, N_('Safemode rendering')),
         self.set('manage_puppetca', N_("Foreman will automate certificate signing upon provision of new host"), true, N_('Manage PuppetCA')),
         self.set('ignore_puppet_facts_for_provisioning', N_("Stop updating IP address and MAC values from Puppet facts (affects all interfaces)"), false, N_('Ignore Puppet facts for provisioning')),
+        self.set('ignore_interfaces_prefixed_with', N_("Ignore interfaces prefixed with these values during facts importing, you can use * wildcard to match names with indexes e.g. macvtap*"), ['lo', 'usb*', 'vnet*', 'macvtap*'], N_('Ignore interfaces prefixed with')),
```

Comment:
Since we allow users to input any kind of expression, I'd not use `prefixed_with` anymore. Something like `ignore_interface_names` would be sufficient.